rubicon-htb.js

```diff
@@ -31,7 +31,7 @@ var RenderService;
 
 //? if (DEBUG) {
 var ConfigValidators = require('config-validators.js');
-var Inspector = require('schema-inspector.js');
+var Inspector = require('schema-inspector');
```

Comment:
Please reverse this change.

Reply:
From NPM doc and https://github.com/Atinux/schema-inspector it seems to be require('schema-inspector');

Reply:
Hi, this is just something that we need internally, is this causing some problem when you are testing?